src/containers/Tenant/Diagnostics/TopQueries/RunningQueriesData.tsx

```diff
@@ -24,32 +22,42 @@ interface Props {
 export const RunningQueriesData = ({database, onRowClick, rowClassName}: Props) => {
     const [autoRefreshInterval] = useAutoRefreshInterval();
     const filters = useTypedSelector((state) => state.executeTopQueries);
-    const {currentData, isLoading, error} = topQueriesApi.useGetRunningQueriesQuery(
+
+    const {tableSort, handleTableSort, backendSort} = useRunningQueriesSort();
+
+    const {currentData, isFetching, error} = topQueriesApi.useGetRunningQueriesQuery(
         {
             database,
             filters,
+            sortOrder: backendSort,
         },
         {pollingInterval: autoRefreshInterval},
     );
 
+    const loading = isFetching && currentData === undefined;
```

Comment:
`isLoading` query param isn't false when data initially loaded, but filters change. So when we change sort order and load new data, there is `no-data` message instead of loader. `isFetching && currentData === undefined;` prevents it